datasets/sentinel-5p/summarize-wf.yaml

```diff
@@ -0,0 +1,75 @@
+name: Summarize Sentinel 5P L2 Items
+
+dataset:
+  owner: microsoft
+  name: sentinel-5p-l2
+
+args:
+- registry
+
+jobs:
+  create-splits:
+    tasks:
+    - id: list-prefixes
+      image: ${{ args.registry }}/pctasks-task-base:rob
+      task: pctasks.task.common.list_prefixes:task
+      args:
+        src_uri: blob://sentinel5euwest/sentinel-5p-stac
+        depth: 4
+      environment:
+        AZURE_TENANT_ID: ${{ secrets.task-tenant-id }}
+        AZURE_CLIENT_ID: ${{ secrets.task-client-id }}
+        AZURE_CLIENT_SECRET: ${{ secrets.task-client-secret }}
+      schema_version: 1.0.0
+  summarize-map:
+    foreach:
+      items: ${{ jobs.create-splits.tasks.list-prefixes.output.uris }}
+    tasks:
+    - id: list-files
+      image: ${{ args.registry }}/pctasks-task-base:latest
+      task: pctasks.task.common.list_files:task
+      args:
+        src_uri: ${{ item }}
+        extensions:
+          - .json
+      environment:
+        AZURE_TENANT_ID: ${{ secrets.task-tenant-id }}
+        AZURE_CLIENT_ID: ${{ secrets.task-client-id }}
+        AZURE_CLIENT_SECRET: ${{ secrets.task-client-secret }}
+      schema_version: 1.0.0
+    - id: summarize-map
+      image: ${{ args.registry }}/pctasks-task-base:rob
+      task: pctasks.task.common.summarize:map_task
+      args:
+        uris: ${{ tasks.list-files.output.uris }}
+        include_keys:
+          - assets.data.description
+          - properties.s5p:product_type
+        summary_settings:
+          max_mixed_value_samples: 100
```

Comment:
Do you think 100 would be a good default? Or is sentinel-5p especially mixed?

Reply:
Yeah, this was on iteration 2 of exploring the dataset, where I needed to capture all the distinct values of the product type and descriptions to catch them all (I think the end list was about 20 values).

I can set it back to the original workflow that is a better default.